changelog/pending/20250823--cli-state--add-pulumi-state-taint-untaint-commands-to-force-replacement-of-resources-upon-next-update.yaml

```
@@ -0,0 +1,4 @@
+changes:
+- type: feat
+  scope: cli/state
+  description: Add pulumi state taint/untaint commands to force replacement of resources upon next update.
```

```suggestion
  description: Add pulumi state taint/untaint commands to force replacement of resources upon next update
```

See https://github.com/pulumi/pulumi/blob/master/CONTRIBUTING.md#changelog-messages